android-json-form-wizard/src/test/java/com/vijay/jsonwizard/widgets/EditTextFactoryTest.java

```diff
@@ -226,6 +228,51 @@ public void testNameWithCumulativeTotalValidatorEditTextFactoryInstantiatesViews
         Assert.assertEquals(1, viewList.size());
     }
 
+    @Test
+    public void testCatchJsonExceptionViewsShouldBeEmpty() throws Exception {
```

Comment:
What is being tested here?

Reply:
verify view list size in case of exception occur.

Reply:
what part of the code changes does this test?

Reply:
https://github.com/opensrp/opensrp-client-native-form/blob/3cd8d23b50318285a3da7887307cab7c91c235a0/android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/EditTextFactory.java#L122

Reply:
this hasn't changed, what changed is that we moved the code into the executors. if that causes a drop in coverage, then we should test that part in particular.

Reply:
yes, i have added this because of coverage dropped.